testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -1430,6 +1609,92 @@ class PlatformViewScrollingUnderWidget extends Scenario
   }
 }
 
+/// Builds a scenario where many platform views with clips scrolling.
+class PlatformViewsWithClipsScrolling extends Scenario
```

Comment:
This is the test added for the crash. It mimics the behavior of the benchmark test that originally caught the crash.